examples/image-streamer/api300/artifact_bundle.rb

```diff
@@ -0,0 +1,73 @@
+# (C) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# You may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative '../../_client_i3s' # Gives access to @client
+
+# Example:
+# - Create, update, download, upload, extract and delete an artifact bundle for an API300 Image Streamer
+# - Create, download, upload an backup bundle for an API300 Image Streamer
+# NOTE: It is needed a DeploymentGroup and a PlanScript previously created
+
+deployment_group = OneviewSDK::ImageStreamer::API300::DeploymentGroup.get_all(@client).first
+plan_script = OneviewSDK::ImageStreamer::API300::PlanScript.get_all(@client).first
+
+options = {
+  name: 'ArtifactBundle Name',
+  description: 'Description of ArtifactBundle'
+}
+
+puts "\nCreating an artifact bundle with plan script"
+item = OneviewSDK::ImageStreamer::API300::ArtifactBundle.new(@client, options)
+item.add_plan_script(plan_script, false)
+item.create
+puts "Artifact Bundle with name #{item['name']} and uri #{item['uri']} created successfully."
+puts 'Data:', item.data
+
+puts "\nUpdating the name of the artifact bundle"
+item.update_name("#{item['name']}_Updated")
+puts "Artifact Bundle with name #{item['name']} and uri #{item['uri']} updated successfully."
+
+puts "\nListing all artifact bundles"
+all_items = OneviewSDK::ImageStreamer::API300::ArtifactBundle.get_all(@client)
+all_items.each { |each_item| puts each_item['name'] }
+
+download_path = '/tmp/artifact-bundle.zip'
```

Comment:
It would be good to be parameterized. Just as it was done in the Golden Image [here](https://github.com/HewlettPackard/oneview-sdk-ruby/blob/master/examples/_client_i3s.rb.example#L35).